Fix grey color on white background

brianhartsock opened the pull request:
Part of #105 

Initial thoughts on how to clean up our grey usage.  This isn't complete yet.   Please review and let me know if you have any concerns or ideas.

Concerns:
* 333 and 666 aren't that easy to distinguish.  See the hover state.
* Zero state title is pretty bold now.  We could make it the same color as the text if we wanted.

Table Headers
![2014-10-23_11-23-01](https://cloud.githubusercontent.com/assets/309711/4755901/337657b8-5ac9-11e4-8209-83c6fc3d0389.png)

Zero State (the header seems a little dark to me now)
![2014-10-23_11-24-11](https://cloud.githubusercontent.com/assets/309711/4755906/3da1e6b2-5ac9-11e4-8d78-3a1e1e330cf6.png)

Loading
![2015-01-09_16-07-07](https://cloud.githubusercontent.com/assets/309711/5687340/943392fe-9819-11e4-9071-58fe2ef3c159.png)

Subtext
![2015-01-09_16-10-59](https://cloud.githubusercontent.com/assets/309711/5687383/20067256-981a-11e4-914e-0eb109ad37fb.png)

Progress subtext
![2014-10-23_11-26-15](https://cloud.githubusercontent.com/assets/309711/4755928/62ee6968-5ac9-11e4-972e-3f603f653799.png)

araiford commented:
Here are the darker grey cloud loading icons:

normal:
![](http://04d4d798c244fdc108be-f47968c0b676e3276188abc4afc95d37.r31.cf1.rackcdn.com/clouding-666.gif)

retina:
![](http://04d4d798c244fdc108be-f47968c0b676e3276188abc4afc95d37.r31.cf1.rackcdn.com/clouding-666@2x.gif)

brianhartsock commented:
I've rebased and updated the description with the latest screenshots.  Couple things to note:

I think we should create the following issues for follow up
* Investigate Table Hover Contrast
* Color Theory Improvements
* Disabled Status Indicator Contrast (as @annwa pointed out in #105)
* Improve Color Management in code - Still lots of grey's that are hard coded, could use lighten/darken functions to reduce number of hard coded colors if that makes sense.

If I get a +1, I can rebase again to squash the commits then merge.

mazzifer commented:
@brianhartsock: LGTM

bradgignac commented on lib/stylesheets/patterns/_list-table.scss:
```diff
@@ -245,17 +245,17 @@ td {
-  color: $grey300;
+  color: $grey500;
   &:before {
     background: image-url('clouding.gif') center center no-repeat;
```
Needs `background-size`.

brianhartsock commented:
@bradgignac fixed

bradgignac commented:
:shipit:

bradgignac commented:
@brianhartsock I'm waiting on an outstanding PR from @riltsken. Once that's in, I'll cut a release.
